apis/duck/v1alpha1/channelable_types_test.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/knative/pkg/apis/duck"
+)
+
+func TestChannelImplementsChannelable(t *testing.T) {
+	duck.VerifyType(&Channel{}, &Channelable{})
```

Comment:
Maybe `VerifyType` should return an error that we can check and call `t.Error` on, rather than panicking as it does today.

Reply:
@ImJasonH That's a good point :+1:

Reply:
That's gonna make the change way bigger though… Should I do that in the same PR or create a follow-up / different PR for that change ?

Reply:
Let's do it all at once

Reply:
ok fair enough, I'll work on that :wink: